artemis/src/test/java/tech/pegasys/artemis/state/util/ValidatorsUtilTest.java

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.state.util;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static tech.pegasys.artemis.Constants.ACTIVE_PENDING_EXIT;
+import static tech.pegasys.artemis.Constants.EXITED_WITHOUT_PENALTY;
+
+import tech.pegasys.artemis.datastructures.beaconchainstate.ValidatorRecord;
+import tech.pegasys.artemis.datastructures.beaconchainstate.Validators;
+import tech.pegasys.artemis.ethereum.core.Hash;
+import tech.pegasys.artemis.util.uint.UInt64;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class ValidatorsUtilTest {
+    public static double DOUBLE_ASSERTION_DELTA = 0.0d;
+
+    @Test
+    public void assert_Get_Zero_As_Effective_Balance_For_NullValidators() {
```

Comment:
We'll write up an issue for ValidatorUtil, as that was an already existing file, but for any new files we should use camelCase naming convention instead of snake_case for our function names and field names (constants excluded). Please see https://google.github.io/styleguide/javaguide.html#s5-naming for reference on how we prefer naming.